fetch/range/blob.any.js

```diff
@@ -196,13 +193,15 @@ supportedBlobRange.forEach(({ name, data, type, range, content_length, content_r
   }, name);
 });
 
-unsupportedBlobRange.forEach(({ name, data, type, headers }) => {
+unsupportedBlobRange.forEach(({ name, data, type, range }) => {
   promise_test(t => {
     const blob = new Blob(data, { "type" : type });
     const blobURL = URL.createObjectURL(blob);
     t.add_cleanup(() => URL.revokeObjectURL(blobURL));
     const promise = fetch(blobURL, {
-      "headers": headers,
+      "headers": {
+        "Range": range
```

Comment:
This definitely simplifies things. I think I originally had a test that added some other headers, but dropped it before the merge.